wordpress-mobile/WordPress-iOS
# Update screenshot generation

Makes the necessary updates for the screenshot generation process to work with the new login and Aztec.

**To test:**
Enter the credentials on `WordPressScreenshotGeneration.swift` and run it, check that it passes.
You can also edit the `Snapfile` to generate only screenshots for one language and one device and run the `snapshot` command.

**Needs review:** @astralbodies 
cc: @nheagy @aerych @diegoreymendez @jleandroperez
